zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -26,6 +32,19 @@ case class Response[-R, +E] private (
   def addCookie(cookie: Cookie): Response[R, E] =
     self.copy(headers = self.headers ++ List(Header.custom(HttpHeaderNames.SET_COOKIE.toString, cookie.encode)))
 
+    def defaultHealders: Response[R,E] = self.headers.map(_.name.toString.toLowerCase()).filter(_=="content-length") match {
+      case _ :: _ => self
+  
+      case immutable.Nil =>self.data match {
+          case Empty => copy(headers=self.headers ++ List(Header("Content-Length",{0}.toString())))
+          case Text(text, _) =>copy(headers=self.headers ++ List(Header("Content-Length",{text.getBytes().length}.toString())))
+          case Binary(data) =>copy(headers=self.headers ++ List(Header("Content-Length",{data.toVector.length}.toString())))
+          case BinaryN(data) =>copy(headers=self.headers ++ List(Header("Content-Length",{data.array().length}.toString())))
+          case BinaryStream(_) =>copy(headers=self.headers ++ List(Header("Transer-Encoding","chunked")))
+        }
+     
+    }  
```

Comment:
Also, I am not sure if you need this. Can you mention the purpose of this change? otherwise remove it

Reply:
changes required for  #448

Reply:
There already exists the method that is suitable for #448:
```
def setPayloadHeaders: Response[R, E] = {
    getContentLength match {
      case Some(value) => setContentLength(value)
      case None        => setChunkedEncoding
    }
  }
```
So, I'm not sure we have to do the same job twice.